packages/rrweb/src/utils.ts

```diff
@@ -670,3 +670,28 @@ export function clearTimeout(
 ): ReturnType<typeof window.clearTimeout> {
   return getImplementation('clearTimeout')(...rest);
 }
+
+/**
+ * Takes a styles attribute string and converts it to key value pairs
+ * @param styles - The full stype attributes string
+ *
+ */
+export function splitStyleAttributes(styles: string) {
+  const splitStyles = styles.split(';');
+  return splitStyles
+    .filter((value) => value.split(':').length == 2)
```

Comment:
Q: Is this safe? Could styles not include colons in other places? Not 100% sure, just want to double check!

Reply:
That will probably be an issue sooner or later, you're right